things/redis/things.go

```diff
@@ -16,6 +17,9 @@ const (
 	idPrefix  = "thing"
 )
 
+// ErrThingRedis indicates error in redis
+var ErrThingRedis = errors.New("Thing Redis error")
```

Comment:
It would be good to know more about this error. Maybe also which operation failed.